packages/api-clients/open-api/bffApi.yml

```diff
@@ -3495,6 +3495,85 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
 
+  /eservices/{eServiceId}/templates/{eServiceTemplateId}/{eServiceTemplateVersion}/interface:
```

Comment:
There should be no need to put the `eServiceTemplateId` and `eServiceTemplateVersionId` in the url.
Those ids could be retrived from the e-service and descriptor itself

Reply:
I would change it to `POST /eservices/{eServiceId}/descriptors/{descriptorId}/instances/interface`

Reply:
related to [PR](https://github.com/pagopa/interop-be-monorepo/pull/1590)